.github/workflows/workflows.yaml

```diff
@@ -311,6 +311,174 @@ jobs:
           manifest-path: "flatpak/web/org.nanuc.Axolotl.yml"
           branch: "main"
 
+  build-axolotl-deb-arm64:
+    name: Build axolotl Debian arm64
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Go 1.16
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: Set up cross-compiling environment
+        run: |
+          sudo apt update
+          sudo apt install -y gcc-aarch64-linux-gnu
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Download dependencies
+        run: go mod download
+
+      - name: Build
+        run: env GOOS=linux GOARCH=arm64 CGO_ENABLED=1 CC=aarch64-linux-gnu-gcc PKG_CONFIG_PATH=/usr/lib/aarch64-linux-gnu/pkgconfig go build -o build/linux-arm64/axolotl .
+
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: linux-arm64
+          path: build/linux-arm64/axolotl
+          retention-days: 1
+
+  build-axolotl-web-deb-arm64:
+    name: Build axolotl-web Debian arm64
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Node 16
+        uses: actions/setup-node@v2.5.0
+        with:
+          node-version: "16.x"
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Create folder
+        run: mkdir --parents $GITHUB_WORKSPACE/build/linux-arm64/axolotl-web
+
+      - name: Download dependencies
+        run: npm --prefix ./axolotl-web --target_arch=arm64 ci --loglevel verbose
```

Comment:
For `build-axolotl-web` we do not specify the loglevel, is there a specific reason to do so here?
I would vote to keep the two build jobs in sync (its just logging, but we are introducing a certain level of duplication here)

Reply:
In the contrary, in line 55, there is the --loglevel defined with verbose. That's where this part for arm64 is from.

Reply:
How we specify the loglevel in the end isnt so important, I will just resolve this comment.